bandit/bandit.go

```diff
@@ -34,6 +30,9 @@ type BanditDialer struct {
 	dialers      []Dialer
 	bandit       *bandit.EpsilonGreedy
 	statsTracker stats.Tracker
+	// lastDialCountryCode stores the last dialed country code.
+	// This is used to avoid unnecessary calls to the UI thread
+	lastDialCountryCode atomic.Value
```

Comment:
Could you take a look at it here, Does this break any other code?
cc @atavism @reflog